src/Microsoft.ML.Data/Transforms/NAFilter.cs

```diff
@@ -26,6 +26,7 @@
 
 namespace Microsoft.ML.Runtime.Data
 {
+    /// <include file='doc.xml' path='doc/members/member[@name="NAFilter"]'/>
```

Comment:
💭 Seems strange that these are in an external file instead of defined here in code...

Reply:
Need them in more than one place. 

---
In reply to: [202774055](https://github.com/dotnet/machinelearning/pull/529#discussion_r202774055) [](ancestors = 202774055)